apps/editor.planx.uk/src/pages/Team/components/FlowTable.tsx

```diff
@@ -0,0 +1,289 @@
+import { useFlowSortDisplay } from "./hooks/useFlowSortDisplay";
+
+const StyledTable = styled(Table)(({ theme }) => ({
+  marginTop: theme.spacing(2),
+  zIndex: 1,
+  position: "relative",
+  border: `1px solid ${theme.palette.border.main}`,
+  borderBottom: 0,
+  borderCollapse: "separate",
+  "& .MuiTableCell-root": {
+    padding: theme.spacing(1.5, 2),
+    borderBottom: `1px solid ${theme.palette.border.main}`,
+  },
+}));
+
+const StyledTableHead = styled(TableHead)(({ theme }) => ({
+  "& .MuiTableCell-head": {
+    backgroundColor: theme.palette.background.default,
+    borderBottom: `1px solid ${theme.palette.border.main}`,
+    position: "sticky",
+    top: 0,
+    zIndex: 2,
+    "&:last-of-type": {
+      borderLeft: `1px solid ${theme.palette.border.main}`,
+    },
+  },
+}));
+
+const StyledTableRow = styled(TableRow, {
+  shouldForwardProp: (prop) => prop !== "isTemplated",
+})<{ isTemplated?: boolean }>(({ theme, isTemplated }) => ({
+  backgroundColor: isTemplated
+    ? theme.palette.template.light
+    : theme.palette.background.default,
+  cursor: "pointer",
+  "&:hover": {
+    backgroundColor: isTemplated
+      ? theme.palette.template.main
+      : theme.palette.background.paper,
+    "& a": {
+      textDecoration: "underline",
+    },
+  },
+  "&:has(.actions-cell:hover) a": {
+    textDecoration: "none",
+  },
+}));
+
+const FlowTitleCell = styled(TableCell)(() => ({
+  width: "45%",
+  minWidth: "240px",
+}));
+
+const FlowStatusCell = styled(TableCell)(() => ({
+  width: "11%",
+  minWidth: "150px",
+}));
+
+const FlowActionsCell = styled(TableCell)(({ theme }) => ({
+  width: "5%",
+  maxWidth: "100px",
+  backgroundColor: theme.palette.background.paper,
+  borderLeft: `1px solid ${theme.palette.border.main}`,
+}));
+
+const FlowLink = styled(Link)(({ theme }) => ({
+  textDecoration: "none",
+  color: theme.palette.text.primary,
+  fontWeight: FONT_WEIGHT_SEMI_BOLD,
+  "&:hover": {
+    textDecoration: "underline",
+  },
+}));
+
+interface FlowTableProps {
+  flows: FlowSummary[];
+  teamId: number;
+  teamSlug: string;
+  refreshFlows: () => void;
+}
+
+export const FlowTable: React.FC<FlowTableProps> = ({
+  flows,
+  teamSlug,
+  refreshFlows,
+}) => {
+  const { showPublished, headerText } = useFlowSortDisplay();
+
+  return (
+    <StyledTable>
+      <StyledTableHead>
+        <TableRow>
+          <FlowTitleCell>Flow title</FlowTitleCell>
+          <FlowStatusCell>Online status</FlowStatusCell>
+          <FlowStatusCell>Flow type</FlowStatusCell>
+          <TableCell>{headerText}</TableCell>
+          <FlowActionsCell align="center">Actions</FlowActionsCell>
+        </TableRow>
+      </StyledTableHead>
+      <TableBody>
+        {flows.map((flow) => (
+          <FlowTableRow
+            key={flow.slug}
+            flow={flow}
+            teamSlug={teamSlug}
+            refreshFlows={refreshFlows}
+            showPublished={showPublished}
```

Comment:
nit: Rather than prop drill these values in, we can just call `useFlowSortDisplay()` within the child.